editor/src/components/canvas/controls/select-mode/select-mode-hooks.ts

```diff
@@ -264,12 +267,17 @@ function useStartDragState(): (
 
       const isTargetSelected = selectedViews.some((sv) => TP.pathsEqual(sv, target))
 
-      // FIXME: Re-establish filtering based on `selectElementsThatRespectLayout`.
-      // https://github.com/concrete-utopia/utopia/pull/1088/files/460e68fb6b4156833744a5441c9e56d662c8b51d#r614029462
       const selection =
         isTargetSelected && TP.areAllElementsInSameScene(selectedViews) ? selectedViews : [target]
 
-      let originalFrames = getOriginalCanvasFrames(selection, componentMetadata)
+      const moveTargets = selection.filter(
+        (view) =>
+          TP.isScenePath(view) ||
+          TP.isStoryboardDescendant(view) || // FIXME This must go in the bin when we separate the Scene from the component it renders
```

Comment:
These two lines still need replacing, but I don't think that should happen in this PR. The reason for this filtering was to enable dragging of actual `Scene` elements, which this combination no longer captures. I'll add a follow up ticket for that.

Reply:
In fact, I think the `Scene` elements would be included in the `elementsThatRespectLayout` since I added the relevant property controls for them. If that's the case, please can you delete the `TP.isScenePath(view) || TP.isStoryboardDescendant(view)` part of the condition.

Reply:
Fixed in 3455633c.